src/shared/main-func.h

```diff
@@ -6,13 +6,15 @@
 #include "sd-daemon.h"
 
 #include "argv-util.h"
+#include "hashmap.h"
 #include "pager.h"
 #include "selinux-util.h"
+#include "signal-util.h"
 #include "spawn-ask-password-agent.h"
 #include "spawn-polkit-agent.h"
 #include "static-destruct.h"
 
-#define _DEFINE_MAIN_FUNCTION(intro, impl, ret)                         \
+#define _DEFINE_MAIN_FUNCTION(intro, impl, result_to_exit_status, result_to_return_value) \
```

Comment:
Hmm, why use different values for EXIT_STATUS= notification and real exit code? And pid1 doesn't really make use of EXIT_STATUS= anyway 🤔